src/spikeinterface/preprocessing/decimate.py

```diff
@@ -93,22 +90,26 @@ class DecimateRecordingSegment(BaseRecordingSegment):
     def __init__(
         self,
         parent_recording_segment,
-        resample_rate,
+        decimated_sampling_frequency,
```

Comment:
yes, but it's soo deep in the API that I'm 100% sure it wouldn't affect anyone's workflow.

If it were at the `DecimateRecording` level, than we should have worried about back-compatibility because of saved objects/JSON files. But since it's the segment which is instantiated on the fly we don't have to worry about it (and I think the naming is much more in line with the overll API)